conductr_cli/conduct_acls.py

```diff
@@ -77,10 +92,19 @@ def tcp_port_value(line):
     for row in data:
         log.screen(
             '{tcp_port: <{tcp_port_width}}{padding}'
+            '{system: <{system_width}}{padding}'
+            '{system_version: <{system_version_width}}{padding}'
+            '{endpoint_name: <{endpoint_name_width}}{padding}'
             '{bundle_id: <{bundle_id_width}}{padding}'
             '{bundle_name: <{bundle_name_width}}{padding}'
             '{status: <{status_width}}'.format(**dict(row, **column_widths)).rstrip())
 
+    duplicate_endpoints = find_duplicate_endpoints(tcp_acls)
+    if duplicate_endpoints:
+        log.screen('')
+        log.warning('Multiple endpoints found: {}'.format(', '.join(duplicate_endpoints)))
+        log.warning('Unable to generate proxy configuration for these endpoints.')
```

Comment:
If the endpoint is duplicated, we won't expose the endpoint through the proxy. Maybe `Unable to expose these endpoint via ConductR HAProxy` is a better error message?